### PR TITLE
add test for add remove

### PR DESCRIPTION
**Pull request by Itswali**
# In This PR.
* Add test to check the functionality of an add to-do list function.
* Add a test to check the functionality for removing a task from the to-do list.

### CODE REVIEW

**Review by FrederickMih — COMMENTED** [review]
Hi @Itswali  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Let's kindly update our README file using this new [README](https://github.com/microverseinc/readme-template/edit/master/README.md) template.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Itswali  :wave: ,
I am Nemwel your reviewer for this PR :man_beard: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## :heavy_multiplication_x:  Invalid Code Review Request :heavy_multiplication_x: 

As this is a pair programming task, kindly ensure that your other two members have made meaningful contribution to the project before re-submiting for review.

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/test/add-remove-todo.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    todo.deleteTodo();
+  });
+});
+
+describe('Test remove functionality', () => {
+  test('Remove todo task selected by the user', () => {
+    const removeTask = new Todo('ensure project test work without errors!');
+    removeTask.deleteTodo();
+    expect(removeTask.getAllTodos().length).toBe(0);
+  });
```
Comment (ichala) [inline src/test/add-remove-todo.test.js]: - [ ] your remove test is not passing , kindly fix the issue 🙏 


image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Correct test file names ✔️ 
- No Linter errors ✔️ 



 ## Required Changes ♻️

- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi TEAM  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — COMMENTED** [review]
[# STATUS: APPROVED :green_circle: :clap: 

Hello  @TEAM    , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- [x]  Your repo is professional 
- [x] Gitflow is followed
- [x] Descriptive P.R
- [x] Descriptive README file




 

- [x] No Linter Errors







Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]

**Pull request merged**